charts/toffee-frontend/values.yaml

```diff
@@ -1,7 +1,6 @@
 nodejs:
   applicationPort: 1337
   image: 'sdshmctspublic.azurecr.io/toffee/frontend:latest'
-  aadIdentityName: toffee
```

Comment:
Apple one needs to go too https://github.com/hmcts/sds-toffee-frontend/blob/b0275f073aff883d28d826877705622b03f13b35/charts/apple-frontend/values.yaml#L4